examples/cactus-check-connection-ethereum-validator/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "check-connection-ethereum-validator",
+  "private": true,
+  "scripts": {
+    "start": "node ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "debug": "nodemon --inspect ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "build": "npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path",
+    "build-ts": "tsc -p ./tsconfig.json",
+    "tslint": "tslint -c tslint.json -p tsconfig.json './*.ts'",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "copy-blp-config": "ts-node copyBLPConfig.ts",
+    "replace-blp-config-path": "ts-node replaceBLPConfigPath.ts",
+    "init-check-connection-ethereum-validator": "ln -s ../examples/cactus-check-connection-ethereum-validator/node_modules ../../dist/node_modules"
+  },
+  "dependencies": {
+    "@types/node": "^14.0.24",
+    "body-parser": "^1.19.0",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "ethereumjs-common": "^1.5.1",
+    "ethereumjs-tx": "^2.1.2",
+    "express": "~4.16.1",
+    "fabric-ca-client": "~1.4.0",
+    "fabric-network": "~1.4.0",
+    "http-errors": "~1.6.3",
+    "jsonwebtoken": "^8.5.1",
+    "log4js": "^3.0.6",
+    "morgan": "~1.9.1",
+    "shelljs": "^0.8.4",
+    "socket.io": "^2.0.4",
+    "ts-node": "8.9.1",
+    "web3": "^1.2.9",
+    "xmlhttprequest": "^1.8.0"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.31.1",
+    "@typescript-eslint/parser": "^4.31.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "^2.4.0",
+    "tslint": "6.0.0",
+    "typescript": "^3.9.3"
```

Comment:
@stepniowskip Is this meant to be a package that is part of the monorepo (and its build/test/release cycle)?

Reply:
Yes, but it will not be tested during CI cycle (for performance reasons)

Reply:
@StepniowskiP Got it, thank you. Do you mean that the CI would take too much longer than it is now or something else? Do you have some rough estimate on how long the tests here take to execute?

Reply:
As far as I know, these tests are supposed to be run on fixed environment just to check connection between verifier and ethereum ledger. From my perspective there is no need to run this tests in CI cycle because whole cycle is taking a lot of time to execute and tests are meant to be used on demand.

Reply:
@StepniowskiP Understood. Do you have some rough estimate on how long the tests here take to execute? For example would you guess it to be closer to 10 minutes or an hour?

Reply:
@petermetz For now tests take about 2-5 minutes on my setup, but there will be more of them as I am going to issue new PR with requested, additional tests.

Reply:
> @petermetz For now tests take about 2-5 minutes on my setup, but there will be more of them as I am going to issue new PR with requested, additional tests.

> As far as I know, these tests are supposed to be run on fixed environment just to check connection between verifier and ethereum ledger. From my perspective there is no need to run this tests in CI cycle because whole cycle is taking a lot of time to execute and tests are meant to be used on demand.

@StepniowskiP Got it. This is just a recommendation, not a change request: Either 
a) The CI should include the tests so that every change made is verified automatically (this is my preference) 
b) Manual testing should be performed for every single commit (mirroring the automation described in a))

My personal guiding principle is this: I'd rather have a slow CI process that is fully automated then a quicker but manual process because the most limited resource we'll always have is contributor brain power and it's a waste using it for repeated manual testing IMO.